apps/studio/src/migration/dev-1.js

```diff
@@ -44,6 +44,14 @@ export default {
       password: 'Example@1',
       defaultDatabase: 'sakila'
     },
+    {
+      name: "[DEV] Docker Bigquery",
+      connectionType: 'bigquery',
+      port: 9050,
+      host: 'localhost',
+      projectId: 'bks',
+      defaultDatabase: 'world',
+    },
```

Comment:
This should be in it's own migration, look at dev-3.js on how to do that. The reason is, for existing development builds this won't run.